app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -41,26 +54,37 @@ export default function SearchResult({ user }: { user: User.AsObject }) {
       <Card className={classes.card}>
         <CardActionArea>
           <CardContent>
-            <div className={classes.resultHeader}>
-              <Avatar user={user} />
-              <Container>
-                <Typography variant="h2">{user.name}</Typography>
-                <Typography variant="subtitle1">
-                  {hostingStatusLabels[user.hostingStatus]}
-                </Typography>
-              </Container>
-            </div>
-
-            <ScoreBar value={user.communityStanding * 100}>
-              {COMMUNITY_STANDING}
-            </ScoreBar>
-            <ScoreBar value={user.verification * 100}>
-              {VERIFICATION_SCORE}
-            </ScoreBar>
-
-            {user.aboutMe.length < 300
-              ? user.aboutMe
-              : user.aboutMe.substring(0, 300) + "..."}
+            <UserSummary user={user}>
+              <div className={classes.statusLabelWrapper}>
+                <div>
+                  <CouchIcon />
+                  <Typography
+                    className={classes.statusLabel}
+                    display="inline"
+                    variant="subtitle1"
+                    color="primary"
+                  >
+                    {hostingStatusLabels[user.hostingStatus]}
+                  </Typography>
+                </div>
+                <div>
+                  <LocationIcon />
+                  <Typography
+                    className={classes.statusLabel}
+                    display="inline"
+                    variant="subtitle1"
+                    color="secondary"
+                  >
+                    {meetupStatusLabels[user.meetupStatus]}
+                  </Typography>
+                </div>
+              </div>
+            </UserSummary>
+            <Typography variant="body1" className={classes.about}>
+              {aboutText(user)}
+            </Typography>
+            <RemainingAboutLabels user={user} />
+            <LabelsReferencesLastActive user={user} />
```

Comment:
\*feels bad\* but now this seems like *too many* fields for a search result...

Reply:
Maybe an appropriately named prop that either show's all fields or just a couple? And put the component in storybook so it's somewhat documented

Reply:
Or maybe split the first component into two, one with just Age / Gender and languages, and the rest, with the latter only on the profile page

Reply:
> *feels bad*
ha! No it's fine
> but now this seems like too many fields for a search result...
@andreaowu and I wanted to see whether it would work or not

![image](https://user-images.githubusercontent.com/60824744/111613338-50022100-87d6-11eb-88b2-f48bd172c34f.png)
vs
![image](https://user-images.githubusercontent.com/60824744/111613708-b129f480-87d6-11eb-9c01-4a2d742fc886.png)

I've separated them out now